### ISSUE TITLE
Feature: Toggle between Light and Dark modes

### ISSUE THREAD

**Issue by Muskan624-ai** [Akshay473/forensic-pro-suite#112]
## Problem
The  dashboard currently lacks the feature to toggle between Light and Dark modes. This feature could help improve accessibility as it only uses Dark mode.

## Proposed Solution
Introduce a theme toggling button (such as a clean Sun/Moon icon switch) onto the global Dashboard top navigation header.

Kindly assign this to me if you find it appropriate as part of GSSoC'26.

**Comment by Anushreer22** [comment]
Hi @Akshay473, I would like to work on this issue under GSSoC'26 if it is available. Please assign it to me.

**Comment by Muskan624-ai** [comment]
@Akshay473 If my above issue is appropriate, Kindly do assign it to me under GSSoC'26 as mentioned in the issue.

**Comment by Muskan624-ai** [comment]
@Akshay473 I have successfully implemented and verified the light/dark theme toggle button across the preloader screen, login portal and main dashboard layout.

Kindly review the PR for the same.
